streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregate.java

```diff
@@ -61,14 +60,15 @@ public void enableSendingOldValues() {
     private class KStreamWindowAggregateProcessor extends AbstractProcessor<K, V> {
 
         private WindowStore<K, T> windowStore;
+        private Forwarder<K, T> forwarder;
```

Comment:
This should be `Forwarder<Windowed<K>, T>`. See comment about `Forwarder` above.